great_expectations/core/expectation_suite.py

```diff
@@ -105,7 +107,9 @@ def __init__(  # noqa: PLR0913
         self.meta = meta
         self.notes = notes
 
-        self._store = project_manager.get_expectations_store()
+    @property
+    def _store(self) -> ExpectationsStore:
+        return project_manager.get_expectations_store()
```

Comment:
We do some deepcopying in the validator that doesn't actually copy this attr over (latent bug) - we can remedy this by grabbing it just-in-time through a property